EZP-26219: Add missing Search Engine slots

alongosz opened the pull request:
Status: **Ready for a review**
This PR, related to JIRA issue [EZP-26219](https://jira.ez.no/browse/EZP-26219), adjusts work done in #1779 for 6.5 and up where signals are configured differently.

**TODO**:
- [x] Add configuration per Search Engine for `SwapLocation` Slot.
- [x] Add configuration per Search Engine for `AssignSection` Slot.
- [x] Add configuration per Search Engine for `UpdateContentMetadata` Slot.

andrerom commented:
Note, this becomes proaktive bug fix, so should rebase on 6.4 or at least 6.5 to make sure it becomes part of next maintenance releases.

alongosz commented:
Backporting this to `v6.4` in #1779

andrerom commented:
@alongosz Changed base on this to 6.5, so rebase this when you are back on it ;)

alongosz commented:
Rebased this PR on `6.5`.
Essential changes which apply for `6.5` are in 5750d8e.
For the container to be built without errors I also added Slots introduced in #1779 (5efc791).

andrerom commented:
Merged in https://github.com/ezsystems/ezpublish-kernel/commit/0a0887d2f2fad8c0d3a61eb53a50976d4c2c214d
